src/test/developTest/java/mServer/crawler/sender/arte/ArteVideoDetailsDeserializerTest.java

```diff
@@ -0,0 +1,52 @@
+package mServer.crawler.sender.arte;
+
+import com.google.gson.JsonObject;
+import java.util.Arrays;
+import java.util.Calendar;
+import java.util.Collection;
+import mServer.test.JsonFileReader;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import org.junit.Test;
+import static org.junit.Assert.*;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+@RunWith(Parameterized.class)
+public class ArteVideoDetailsDeserializerTest {
+    
+    @Parameterized.Parameters
+    public static Collection<Object[]> data() {
+        return Arrays.asList(new Object[][] {   
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this array creation and simply pass the elements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3878)